Added more formatting options and changed name (#463)
Codecademy/docs: content/markdown/concepts/emphasis/formating.md

```markdown
This avacado egg sandwich is _very_ good.
```

## Strikethrough 

To make a strikethrough text, use 2 tildes `~` before and after a piece of text.

```markdown
~~Strikethrough text~~
```

For example:

```markdown
I did ~~not~~ build a website.
```

Different formatting options can be combined:

```markdown

> **This is in bold, _italics_, ~~and strikethrough~~**

Reviewer: ```suggestion
**This is in bold, _italics_, ~~and strikethrough~~**
```
```

```markdown
This avacado egg sandwich is _very_ good.
```

## Strikethrough 

> To make text strikethrough, use 2 ~ before and after a peace of text.

Reviewer: ```suggestion
To make text strikethrough, use 2 `~` before and after a piece of text.
```

When we include code in text, we'll wrap it in back ticks "`"

Reviewer: In addition, the "~" is called a tilde, so we can write that out for clarity here ☺️ 
Another point is writing out numbers unless they're meant to be an integer data type, i.e. two instead of 2

> To make text strikethrough, use 2 `~` before and after a piece of text.

Reviewer: ```suggestion
To make a strikethrough text, use 2 tildes `~` before and after a piece of text.
```

```markdown
~~Strikethrough text~~
```

For example:

```markdown

> I did ~~not~~ code a website

Reviewer: ```suggestion
I did ~~not~~ build a website.
```

This is very much up to preference

```

> You can also combine different formatting options.

Reviewer: ```suggestion
Different formatting options can be combined:
```

Avoid second-person pronoun "you"

> 
> Example:

Reviewer: ```suggestion
```

```markdown
**This is bold __and italics__ ~~and strikethrough~~**

> ```

Reviewer: Strongly recommend using [Grammarly](https://app.grammarly.com/) to check for any typos!

> You can also combine diffrent formmating opptions.
> Example:
> 
> ```markdown
> **This is bold __and italics__ ~~and strikethrough~~**
> ```

Reviewer: ```suggestion
You can also combine different formatting options.

Example:

```markdown
**This is in bold, _italics_, ~~and strikethrough~~**
```
```

> **This is in bold, _italics_, ~~and strikethrough~~**

Reviewer: Let's add some closing backticks for this code snippet here.
